domains/corruptedarc.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Ishaan1313",
+           "email": "89083494+Ishaan1313@users.noreply.github.com",
+           "discord": "692993163333337139"
+        },
+    
+        "record": {
+            "CNAME": "phones-sells.gl.at.ply.gg:50105"
```

You need to remove the port numbers so that it will actually be a valid CNAME record
```suggestion
            "CNAME": "phones-sells.gl.at.ply.gg"
```